demos/python_demos/image_inpainting_demo/README.md

```diff
@@ -12,11 +12,12 @@ This demo can work in 2 modes:
 Running the application with the `-h` option yields the following usage message:
 
 ```
-usage: image_inpainting_demo.py [-h] -m MODEL [-i INPUT] [-d DEVICE] [-r]
+usage: image_inpainting_demo.py [-h] -m MODEL [-i INPUT] [-d DEVICE]
```

Comment:
There is explanatory text above this that should be updated with the new option names.